lib/models/instrumentation.js

```diff
@@ -244,7 +244,12 @@ class Instrumentation {
     if (!instr.token) {
       throw new Error(`Cannot stop instrumentation "${name}".  It has not started.`);
     }
-    instr.token.stop();
+    try {
+      instr.token.stop();
+    } catch (e) {
+      this.ui.writeLine(chalk.red(`Error reporting instrumentation '${name}'.  Stack: ${this._heimdall.stack}`));
```

Comment:
should we also include e.stack (so we can actually triage from the output)? Possibly this could just be a logger.error though (if we want to keep the actual console clear of heimdall errors)...